src/accelerate/accelerator.py

```diff
@@ -82,6 +82,9 @@ class Accelerator:
 
             Will default to :obj:`["torch"]` for PyTorch versions <=1.5.1 and :obj:`["generator"]` for PyTorch versions
             >= 1.6.
+        central_dataloader (:obj:`bool`, `optional`, defaults to :obj:`False`):
```

Comment:
I don't like `central` as I don't really understand what it means from a quick read, but I spent 10 minutes trying to find a better name without success, so fine to leave it like it is :)

Reply:
Let's go for `dispath_batches` to look like `split_batches`